OpenTrons/opentrons-api
# fix(app, components): center content in instruments and modules cards

closes [RQA-1488](https://opentrons.atlassian.net/browse/RQA-1488)

# Overview

Instrument, Pipette, and Module Cards shown in InstrumentsAndModules are all styled differently. This PR aligns content on each card for both Flex and OT-2. It may be worthwhile in the future to refactor gripper, pipette, and module cards to an abstract Card component with an image and text content.

# Test Plan

- Check that Flex gripper, pipette, and module card content are aligned 
<img width="1217" alt="Screen Shot 2023-10-03 at 10 21 59 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/22499da9-52fe-4a0a-b105-85893cb7488a">

- Check that OT-2 pipette and module card content are aligned 
<img width="1225" alt="Screen Shot 2023-10-03 at 10 21 34 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/8b79491e-03e7-4f1d-8d3b-181dcc3602be">

# Changelog

- Change padding and image sizing on instrument, pipette, and module cards

# Risk assessment

low

[RQA-1488]: https://opentrons.atlassian.net/browse/RQA-1488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
